### PR TITLE
Provide Quarterly Earnings Estimates Support

### PR DESCRIPTION
**Pull request by Bensign**
As recommended by the IEX Team, provide support for the `estimates` API call for getting earnings information. The original issue I opened with more detail is referenced here: https://github.com/iexg/IEX-API/issues/681

Ruby client users may now call `IEX::Resources::Estimates.get('MSFT')` and have the earnings estimate object returned. In the event this information isn't known or reported yet by IEX, `nil` is returned. 

Please let me know if this PR misses anything - happy to get it cleaned up as need be.

### CODE REVIEW

File: lib/iex/resources/estimates.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      property 'consensus_eps', from: 'consensusEPS'
+      property 'number_of_estimates', from: 'numberOfEstimates'
+      property 'fiscal_period', from: 'fiscalPeriod'
+      property 'fiscal_end_date', from: 'fiscalEndDate'
+      property 'report_date', from: 'reportDate'
+
+      def self.get(stock_symbol)
+        Base.symbol(stock_symbol) do
+          response = IEX::Api::Estimates.get(stock_symbol)
+          return nil if response['estimates'].nil?
```
Comment (dblock) [inline lib/iex/resources/estimates.rb]: Remove `nil`, so `return unless response['estimates']`

File: spec/iex/resources/estimates_spec.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      end
+    end
+  end
+  context 'symbol with no estimate', vcr: { cassette_name: 'estimates/zg' } do
+    subject do
+      IEX::Resources::Estimates.get('ZG')
+    end
+    let(:estimates) { subject }
+    it 'returns nil' do
+      expect(estimates).to eq nil
```
Comment (dblock) [inline spec/iex/resources/estimates_spec.rb]: I wonder whether it should it be `nil` or `[]`? What do we do elsewhere?

**Comment by dblock** [comment]
Bump @Bensign? Also needs to be edited/rebuilt on top of IEX cloud APIs, see #42.

**Comment by Bensign** [comment]
Oh wow - awesome to see #42! I have some time this weekend and will get this updated and resolved. Sorry for the delay on this.

**Pull request closed without merging**